validphys2/src/validphys/core.py

```diff
@@ -750,6 +750,15 @@ def __str__(self):
 
 # TODO: Decide if we want methods or properties
 class Stats:
+    """Class holding statistical information about the object used in validphys
+    By the PDF convention, the central value corresponds to member 0.
+    This is equal to the mean of the error members only for the PDF
+    itself and for linear quantities such as DIS-type observables.
+
+    If you want to obtain the average of the error members you can do:
+    ``np.mean(stats_instance.error_members, axis=0)``
```

Comment:
I still don't really understand why you mention this? Is it just to explain the above in a different way? Since I think the above explanation of "rep0 is defined the average over the error members" is just fine

Reply:
Because np.mean(error_members) is not the same as data[0] which is the central value, for instance in a simple hadronic case we have <f^2> vs <f>^2

Reply:
Okay I see, I was indeed just thinking of PDFs. Maybe it's worth stressing indeed that these "vp objects" can be any function of a PDF set. Even though in principle this explanation makes sense and I agree the misunderstanding is my fault, the "By the PDF convention, the central value corresponds to member 0" made me think of PDFs - probably because I don't really talk about member 0 when it's anything other than PDFs.